airbyte-integrations/connectors/source-auth0/source_auth0/manifest.yaml

```diff
@@ -0,0 +1,115 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+
+  requester:
+    type: HttpRequester
+    url_base: "{{ config['base_url'] }}"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['credentials']['access_token'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```

Comment:
Hi @sajarin is this paginator able to handle auth0 pagination?

there is an open PR (for a bug resolution) about it https://github.com/airbytehq/airbyte/pull/29359 and wondering how the pagination will be handled by the new low-code CDK.
Cheers